src/zc/buildout/tests.py

```diff
@@ -3704,7 +3704,6 @@ def test_suite():
                ])
             ),
         doctest.DocFileSuite('testing_bugfix.txt'),
-        unittest.makeSuite(UnitTests),
```

Comment:
I recommend `unittest.defaultTestLoader.loadTestsFromName(__name__)` instead of manually calling `unittest.makeSuite()` for every TestCase subclass.

(Assuming there are any unittest.TestCase subclasses left.)

Reply:
Nope :-) I just removed it :-)